packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -539,6 +539,16 @@ export interface IContextualMenuItem {
    */
   [propertyName: string]: any;
 
+  /**
+   * Detailed description of the menu item for the benefit of screen readers.
+   */
+  ariaDescription?: string;
+
+  /**
```

Comment:
`ariaDescribedBy` should not be *passed in* to `IContextualMenuItem`. This is because the 'element which describes the item' will the `ContextualMenuButton`'s own `Tooltip`. In fact, `title` already is used for a tooltip, so we really should avoid overloading `ariaDescription` here as well. Between `text`, `ariaLabel`, and `title`, we already have all fields covered.

Reply:
Thanks for the input, sing title for the tooltip is not really accessible and not an option for us. Here is a wiki explaining why you should avoid using title (https://sarahmhigley.com/writing/tooltips-in-wcag-21/) and accessibility issues with browser tooltips.  The current model with text, ariaLabel and title does not support our needs in Office. We have some controls that have a title displayed, and then have additional information in a tooltip and these items can be in context menus. We are currently passing the tooltip to aria-label but that means the actual text in the menu item is not read out which means screen reader users get a different experience than non-screen reader users. Happy to discuss more!

Reply:
@ThomasMichon I may have misunderstood here. Are you suggesting I just remove the ariaDescribedBy passed in and leave the ariaDescription? Also in Office we don't use FluentUI's tooltips for a variety of reasons so we can't rely on the FluentUI Tooltip setting anything for us.